Sources/Common/Concurrency/TaskExtension.swift

```diff
@@ -42,6 +42,29 @@ public extension Task where Success == Never, Failure == Error {
             }
         }
     }
+
+    static func periodic(delay: TimeInterval? = nil,
+                         interval: TimeInterval,
+                         operation: @escaping @Sendable () async throws -> Void,
```

Comment:
A different version of this method that allows operation to throw.  Throwing within tasks cancels them, so this is rather standard.